src/java/org/pantsbuild/tools/junit/impl/ConsoleRunnerImpl.java

```diff
@@ -933,7 +933,7 @@ public void setNumRetries(int numRetries) {
       if (test.startsWith("@")) {
         try {
           String argFileContents = Files.toString(new File(test.substring(1)), Charsets.UTF_8);
-          tests.addAll(Arrays.asList(argFileContents.split("\\s+")));
+          tests.addAll(Arrays.asList(argFileContents.split("\n")));
```

Comment:
Presumably nobody is using this quirk but it could trip up someone. I'm not sure if this should go through a deprecation. That would be tricky to coordinate from the python side where we have the infa for that sort of thing.

Reply:
I could extract this as a configurable thing, which would allow users to revert to the previous behavior.

Reply:
I'm not sure if it's worth it or not though. I searched for maven references to the pants junit jar and only found one outside pants. In pants, the args are constructed as a newline delimited file.

Reply:
SGTM

Reply:
So, after writing a test, I discovered that the argfile parsing code here is never reached because the library has already inlined the argfile and it uses newlines as separators.